### PR TITLE
Results command to show test results summary from CLI

### PR DESCRIPTION
**Pull request by OndraM**
This is CLI equivalent to viewing results.xml file in a browser. Similarly this new `steward results` command could be used either when the `run` command is still in progress or when it is finished.

It supports both verbosity modes - default, showing only testcases, and debug (`-vvv`) showing both testcases and tests. 
### Default non-verbose mode

screenshot_20160430_005255
### Verbose mode

screenshot_20160430_005453
### Verbose mode during run

screenshot_20160430_005541

### CODE REVIEW

File: src/Console/Command/ResultsCommand.php [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $filePath = $input->getOption(self::OPTION_RESULTS_FILE);
+
+        if (!is_readable($filePath)) {
+            throw new \RuntimeException(
+                sprintf(
+                    'Cannot read results file "%s", make sure it is accessible '
+                    . '(or use --results-file option if it is stored elsewhere)',
```
Comment (jirinovak) [inline src/Console/Command/ResultsCommand.php]: Maybe `self::OPTION_RESULTS_FILE` should be used here as well.

File: src/Console/Command/ResultsCommand.php [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+     * @param \DateTimeInterface|null $start
+     * @param \DateTimeInterface|null $end
+     * @param bool $inProgress
+     * @return string
+     */
+    private function formatDiff($start, $end, $inProgress = false)
+    {
+        $output = '';
+
+        if ($inProgress && !$end) { // Is is in progress and end time is not available, use current time
```
Comment (jirinovak) [inline src/Console/Command/ResultsCommand.php]: Typo `It is`.

File: src/Console/Command/ResultsCommand.php [hunk 3]
```diff
@@ -0,0 +1,333 @@
...
+    {
+        $output = $result;
+
+        if ($parentTestcaseResult == ProcessWrapper::PROCESS_RESULT_FATAL
+            && $status == AbstractPublisher::TEST_STATUS_STARTED
+        ) {
+            return '<fg=yellow>fatal</>';
+        }
+
+        $colorMap = [
```
Comment (jirinovak) [inline src/Console/Command/ResultsCommand.php]: This function is called from `foreach` cycle, thus this array should be static...

File: src/Console/Command/ResultsCommand.php [hunk 4]
```diff
@@ -0,0 +1,333 @@
...
+     * @return string
+     */
+    private function formatTestResult($result, $status, $parentTestcaseResult)
+    {
+        $output = $result;
+
+        if ($parentTestcaseResult == ProcessWrapper::PROCESS_RESULT_FATAL
+            && $status == AbstractPublisher::TEST_STATUS_STARTED
+        ) {
+            return '<fg=yellow>fatal</>';
```
Comment (jirinovak) [inline src/Console/Command/ResultsCommand.php]: Yellow for fatal? Imho red will be more accurate.

File: src/Console/Command/ResultsCommand.php (hunk 1 above)
Comment (OndraM) [inline src/Console/Command/ResultsCommand.php]: Ok

File: src/Console/Command/ResultsCommand.php (hunk 3 above)
Comment (OndraM) [inline src/Console/Command/ResultsCommand.php]: Nice catch, and similarly in `formatTestcaseResult()`.

File: src/Console/Command/ResultsCommand.php (hunk 4 above)
Comment (OndraM) [inline src/Console/Command/ResultsCommand.php]: I want to distinguish it from "usual" error, which uses red. I tried few other colors, and it doesn't look much better :). Do you have any suggestion? In current revision I also [deduplicated](https://github.com/lmc-eu/steward/pull/65/files#diff-748943b5f72fe252742e36005a051907R30) the fatal error color definition to one place, so feel free to try some better color combination (maybe with background color?) if interested.

Also note the same color could be used in the results.xml, which currently uses also yellow.

File: src/Console/Command/ResultsCommand.php (hunk 1 above)
Comment (OndraM) [inline src/Console/Command/ResultsCommand.php]: BTW thanks unit tests, I swapped the arguments order while doing this :-)) 
https://travis-ci.org/lmc-eu/steward/jobs/127921608#L321

**Pull request merged**